Assets/Scripts/ROS Scripts/ROS Connections v2/Sensors/LampSensor_ROSSensorConnection.cs

```diff
@@ -77,6 +79,11 @@ void Update()
         }
     }
 
+    public string GetSensorName()
+    {
+        return sensorName;
```

Comment:
why not just return this.gameObject.name? That way you dont run the risk of someone modifying sensorName or gameObject.name without modifying the other?